openmc/deplete/pool.py

```diff
@@ -15,7 +15,7 @@
 NUM_PROCESSES = None
 
 
-def deplete(func, chain, x, rates, dt, matrix_func=None, **func_args):
+def deplete(func, chain, x, rates, dt, matrix_func=None, *func_args):
```

Comment:
The description in the docstring needs to be updated since these are no longer keyword arguments. Also, for clarity, I would call it `matrix_args` instead of `func_args` (which might lead someone to believe they are arguments getting passed to `func`, which is not the case).

Reply:
This should be ok now.

Let me know if there is anything else.